package.json

```diff
@@ -68,10 +73,11 @@
     "commitlint": "^17.0.3",
     "compare-versions": "^5.0.1",
     "conventional-changelog-conventionalcommits": "^5.0.0",
-    "crypto": "^1.0.1",
```

Comment:
Why was the `crypto` removed? Is there no problem with the build?

Reply:
`crypto` is a inline module in `node`. The package is be deprecated.